xbmc/pvr/PVRManager.cpp

```diff
@@ -133,6 +140,19 @@ void CPVRManager::Announce(AnnouncementFlag flag, const char *sender, const char
     TriggerEpgsCreate();
     TriggerTimersUpdate();
   }
+  else if (strcmp(message, "OnSleep") == 0)
+  {
+    for (auto updateJob : m_pendingUpdates)
```

Comment:
you should use `CPVRManager::StopUpdateThreads()` instead and clear pending updates in there too